### PR TITLE
BZ2092705: Changes api_floating_ip to application_floating_ip

### PR DESCRIPTION
**Pull request by xenolinux**
Follow up PR of https://github.com/openshift/openshift-docs/pull/47689
This PR changes api_floating_ip to application_floating_ip

Version(s): 4.6+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2092705

Link to docs preview:  http://file.rdu.redhat.com/sdudhgao/app-floating-ip/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-accessing-api-floating_installing-openstack-installer-custom

Also added a screenshot https://github.com/openshift/openshift-docs/pull/48397#issuecomment-1196898795 of a preview

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
Screenshot 2022-07-27 at 8 49 08 PM

File: modules/installation-osp-accessing-api-floating.adoc [hunk 1]
```diff
@@ -63,12 +63,12 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 ====
 If you do not control the DNS server, you can access the cluster by adding the cluster domain names such as the following to your `/etc/hosts` file:
 
-* `<api_floating_ip> api.<cluster_name>.<base_domain>`
-* `<api_floating_ip> grafana-openshift-monitoring.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> prometheus-k8s-openshift-monitoring.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> oauth-openshift.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> console-openshift-console.apps.<cluster_name>.<base_domain>`
-* `api_floating_ip integrated-oauth-server-openshift-authentication.apps.<cluster_name>.<base_domain>`
+* `<application_floating_ip> api.<cluster_name>.<base_domain>`
```
Comment (kalexand-rh) [inline modules/installation-osp-accessing-api-floating.adoc:66]: Looking at the previous PR, this first line should still be `<api_floating_ip>`.

Comment (xenolinux) [reply modules/installation-osp-accessing-api-floating.adoc]: my bad. Addressed

Comment (kalexand-rh) [reply modules/installation-osp-accessing-api-floating.adoc]: It looks like this matches the terminology in https://github.com/openshift/openshift-docs/commit/392e1350b62bb1d689bbe3799dae79130f3550d9, so I'm going to merge it. Thank you!

**Pull request merged**